client/slack-spring-test-api-client/src/main/kotlin/io/olaph/slack/client/test/group/conversation/MockConversationMethodGroup.kt

```diff
@@ -78,7 +79,7 @@ class MockConversationMethodGroup : ConversationsMethodGroup {
     }
 
     override fun setTopic(authToken: String): ConversationsSetTopicMethod {
```

Comment:
has to return MockMethod

Reply:
wait a minute how can the tests pass if that is wrong